src/scripts.js

```diff
@@ -0,0 +1,20 @@
+window.onload = function(){
+    console.log("loaded")
+    window.location="#!"
+
+    var sourceEl = document.getElementById("view-source")
+    sourceEl.removeAttribute("href");
+    sourceEl.onclick = function(){
+        var client = new XMLHttpRequest();
+        client.open('GET', 'https://hashbang.sh/');
+        client.onreadystatechange = function() {
+            console.log(client.responseText)
+            var div = document.createElement("div");
+            var sourceText = document.createTextNode(client.responseText)
+            div.appendChild(sourceText)
+            div.id = "source"
+            document.getElementsByTagName("html")[0].appendChild(div);
```

Comment:
Instead of `document.getElementsByTagName("html")[0]` why not use `document.firstElementChild`?
But also, shouldn't you be appending to `document.body` anyway?